addons/calendar/tests/test_calendar_recurrent_event_future.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
```

Comment:
good to have python test but can you add a javascript unit test or tour for this change ?